src/aws-metadata.js

```diff
@@ -0,0 +1,81 @@
+import request from 'request';
+import async from 'async';
+import {Logger} from './Logger';
+
+/*
+  Utility class for pulling AWS metadata that Eureka requires when registering as an Amazon instance (datacenter).
+*/
+export class AwsMetadata {
```

Comment:
Yeah I went back and forth on that... 
https://www.approxion.com/?p=303
http://stackoverflow.com/questions/15526107/acronyms-in-camelcase

I could go either way. I'll let @daveedgarcia chime in. :)